.github/scripts/templates/footer.md.jinja

```diff
@@ -0,0 +1,4 @@
+%% Hub footer: Please don't edit anything below this line %%
+# This note in GitHub
+
+<span class="git-footer">[Edit In GitHub](https://github.dev/obsidian-community/obsidian-hub/blob/main/{{ file_path }} "git-hub-edit-note") | [Copy this note](https://raw.githubusercontent.com/obsidian-community/obsidian-hub/main/{{ file_path }} "git-hub-edit-note") | [Download this vault](https://github.com/obsidian-community/obsidian-hub/archive/refs/heads/main.zip "git-hub-edit-note") </span>
```

Comment:
After running the script, it looks like the links still need checking here.

The Edit in GitHub opens the README 
The Copy this note gives a 404

I imagine it would be fine for this to be done after the merge, as the content of this is likely to change..